lib/ae_mdw/db/name.ex

```diff
@@ -245,6 +245,46 @@ defmodule AeMdw.Db.Name do
     |> Enum.into(%{})
   end
 
+  @spec ownership(Model.name()) :: %{current: Format.aeser_id(), original: Format.aeser_id()}
+  def ownership(
+        Model.name(
+          index: plain_name,
+          claims: [{_block_index, last_claim_txi} | _rest_claims],
+          transfers: transfers,
+          owner: owner
+        )
+      ) do
+    Model.tx(id: tx_hash) = read_tx!(last_claim_txi)
+    {:ok, name_hash} = :aens.get_name_hash(plain_name)
+
+    orig_owner =
```

Comment:
When there is no transfer, could it use the `Model.name(:owner)`? This way it would call the Node only there is a transfer.

Reply:
You're right, good call. Done